tests/unittest/_torch/executor/test_router_dealer_ipc.py

```diff
@@ -0,0 +1,373 @@
+import multiprocessing
+import pickle
```

Comment:
You may need to add this to a pre-merge test list, as far as I can see, the dangling test file won't be tested.

Reply:
Unit tests under existed folder will be tested now, thx~